src/Couchbase.Lite.Tests.Shared/LogTest.cs

```diff
@@ -291,6 +291,47 @@ public void TestConsoleLoggingLevels()
             Database.Log.Console.Level = LogLevel.Warning;
         }
 
+        [Fact] void TestSetLogLevel()
+        {
+            WriteLog.To.Database.I("IGNORE", "IGNORE"); // Skip initial message
+            Database.SetLogLevel(LogDomain.All, LogLevel.None);
+            var stringWriter = new StringWriter();
+            Console.SetOut(stringWriter);
+            WriteLog.To.Database.E("TEST", "TEST ERROR");
+            stringWriter.Flush();
+            stringWriter.ToString().Should().BeEmpty("because logging is disabled");
+
+            Database.SetLogLevel(LogDomain.All, LogLevel.Verbose);
```

Comment:
again, why are we keeping this out of the for loop below? is there a specific reason?

Reply:
Different test scenario.

Reply:
Will add log level verbose into the level array.